skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -148,7 +148,7 @@
     <string name="send_instance_title">Send Data</string>
     <string name="bluetooth_instance_name_default">Bluetooth Device</string>
     <string name="tv_sender_wait_for_connect">waiting for connection&#8230; \n will start transfer when connected </string>
-    <string name="tv_form_already_exist">form already exists \n please send different form</string>
+    <string name="tv_form_already_exist">form already exist \n please send different form</string>
```

Comment:
A message should not be printed like asking the user to send a different form, I think `Form already exists at receiver` will give the user enough information about any form transaction.

And this message should be appended with each and every form we send. Lets say if we send 3 forms with name form1, form2, form3 and all forms exists there, then we should be printing
form1- Form already exists at receiver's
form2- Form already exists at receiver's
form3- Form already exists at receiver's
We should give this information for each of the individual form transaction.